src/gracenotegroup.ts

```diff
@@ -90,7 +94,8 @@ export class GraceNoteGroup extends Modifier {
       );
     }
 
-    state.leftShift += groupShift;
+    if (right) state.rightShift += groupShift;
+    if (left) state.leftShift += groupShift;
```

Comment:
I think the changes in lines 78-98 could be improved.

Is the default situation `left == true` and `right == false`? We can never have both left and right have the same value, right?

Why not just have one flag `let left = true; // put the grace note group on the left side`

Then if we ever see Modifier.Position.RIGHT, we set left to false.

Then on lines 97-98, just do one or the other depending on the `left` flag.

Reply:
Hi Ron, I have to look into that. But I guess that theoretically it is possible to have two voices each one with grace notes and one to the right and one to the left.

Reply:
I see. If they aren't mutually exclusive then it's fine to have two separate flags. Can we add a test case that shows a note group with grace notes on both sides?

Reply:
I will add a test case on another PR